### PR TITLE
chore(sample): fix lint and ts sample

### PR DESCRIPTION
**Pull request by freeboub**
start fix eslint and tsc warnings in the sample

### CODE REVIEW

File: examples/basic/package.json
Comment (KrzysztofMoch) [inline examples/basic/package.json:1]: I thought that i remove `react-native-video` in #3353. (to remove dep loop, i used manual linking)

File: examples/basic/android/app/src/main/java/com/videoplayer/MainApplication.java [hunk 1]
```diff
@@ -29,7 +29,6 @@ public boolean getUseDeveloperSupport() {
         protected List<ReactPackage> getPackages() {
           @SuppressWarnings("UnnecessaryLocalVariable")
           List<ReactPackage> packages = new PackageList(this).getPackages();
-          packages.add(new ReactVideoPackage());
```
Comment (KrzysztofMoch) [inline examples/basic/android/app/src/main/java/com/videoplayer/MainApplication.java]: This one should be there for manual linking (see #3353)

File: examples/basic/package.json
Comment (KrzysztofMoch) [reply examples/basic/package.json:1]: Please remove it, it seems like my mistake

**Review by KrzysztofMoch — CHANGES_REQUESTED** [review]

**Comment by freeboub** [comment]
@KrzysztofMoch I backport your CI changes to be able to build the sample in this (Thank you again for that).
I want to enhance it be able to build with different build option (with or without ads typically).
What do you advise ? copy build-<platform>.yml to build-<platform>-withads.yml or directly patch the file with something like:
        run: cd examples/basic/android && ./gradlew assembleDebug --build-cache  && ./gradlew clean && export ENABLE_ADS=true && ./gradlew assembleDebug --build-cache && cd ../../..

**Comment by freeboub** [comment]
tI don't understand why I don't see my initial changes /o\

**Comment by KrzysztofMoch** [comment]
> @KrzysztofMoch I backport your CI changes to be able to build the sample in this (Thank you again for that).
> 
> I want to enhance it be able to build with different build option (with or without ads typically).
> 
> What do you advise ? copy build-<platform>.yml to build-<platform>-withads.yml or directly patch the file with something like:
> 
>         run: cd examples/basic/android && ./gradlew assembleDebug --build-cache  && ./gradlew clean && export ENABLE_ADS=true && ./gradlew assembleDebug --build-cache && cd ../../..

You can create separate "job" in build file so we will have build for Ads and build with non ads

**Review by KrzysztofMoch — APPROVED** [review]
LGTM 👍

**Pull request merged**